monai/transforms/utility/dictionary.py

```diff
@@ -1032,8 +1032,10 @@ class Lambdad(MapTransform, InvertibleTransform):
             each element corresponds to a key in ``keys``.
         inv_func: Lambda/function of inverse operation if want to invert transforms, default to `lambda x: x`.
             It also can be a sequence of Callable, each element corresponds to a key in ``keys``.
-        overwrite: whether to overwrite the original data in the input dictionary with lamdbda function output.
+        overwrite: whether to overwrite the original data in the input dictionary with lambda function output.
```

Comment:
I think it'll be simpler to use `overwrite` as the parameter for the output key. if `overwrite` is `True`, it overwrites the original input `key`, if `overwrite` is a hashable key other than `True`, `d[overwrite]` will store the lambda output

Reply:
Hi @wyli, thanks for the suggestions, I have updated it in https://github.com/Project-MONAI/MONAI/pull/5526/commits/65787202c5760d75448fb3cd5ecb241a3eb1e698. Thanks!